### PR TITLE
i18n: import

### PR DESCRIPTION
**Pull request by connorjclark**

### CODE REVIEW

File: shared/localization/locales/cs.json [hunk 1]
```diff
@@ -480,10 +480,10 @@
     "message": "Čtečky obrazovek nedokážou přeložit obsah jiného typu, než je text. Když k prvkům `<object>` přidáte alternativní text, čtečky obrazovek budou moci předat uživatelům význam. [Další informace](https://web.dev/object-alt/)"
   },
   "lighthouse-core/audits/accessibility/object-alt.js | failureTitle": {
-    "message": "Prvky `<object>` nemají text `[alt]`"
+    "message": "Prvky `<object>` nemají alternativní text"
   },
   "lighthouse-core/audits/accessibility/object-alt.js | title": {
-    "message": "Prvky `<object>` mají text `[alt]`"
+    "message": "Prvky `<object>` mají alternativní text"
```
Comment (connorjclark) [inline shared/localization/locales/cs.json:486]: uh?

Comment (adamraine) [reply shared/localization/locales/cs.json:486]: https://github.com/GoogleChrome/lighthouse/pull/13517

File: shared/localization/locales/lv.json [hunk 2]
```diff
@@ -1758,7 +1764,7 @@
     "message": "Šīs pārbaudes ir vajadzīgas saskaņā ar standarta [PWA kontrolsarakstu](https://developers.google.com/web/progressive-web-apps/checklist), taču Lighthouse neveic tās automātiski. Tās neietekmē jūsu rezultātu, taču ir svarīgi pārbaudīt šos lietotnes aspektus manuāli."
   },
   "lighthouse-core/config/default-config.js | pwaCategoryTitle": {
-    "message": "PWA"
+    "message": "PTL"
```
Comment (adamraine) [inline shared/localization/locales/lv.json:1767]: Is it bad practice to say "leave the whole thing untranslated" in a translator comment?

**Review by adamraine — APPROVED** [review]

File: shared/localization/locales/lv.json (hunk 2 above)
Comment (connorjclark) [reply shared/localization/locales/lv.json:1767]: we should just not have this be a i18n string

Comment (adamraine) [reply shared/localization/locales/lv.json:1767]: I'd be ok to override this once and make the changes after release

**Comment by hamad22h** [comment]
H

**Review by emyanas — COMMENTED** [review]
https://elkemaa.com/2022/01/28/%d9%85%d8%a7%d9%8a%d9%86-%d9%83%d8%b1%d8%a7%d9%81%d8%aa-%d8%a7%d9%84%d8%aa%d8%b9%d9%84%d9%8a%d9%85%d9%8a%d8%a9-%d9%84%d8%b9%d8%a7%d9%85-2022/

**Pull request merged**